cli/src/command/prove.rs

```diff
@@ -55,7 +55,7 @@ pub struct LocalProveArgs {
 #[derive(Debug, Args)]
 pub struct CommonProveArgs {
     /// Build artifacts with the specified profile. "release-unoptimized" is default.
-    #[arg(long, default_value = "release-unoptimized")]
+    #[arg(long, default_value = "release")]
```

Comment:
You also need to modify `cli/src/command/new` to not use the `release-unoptimized` profile in guest projects.

Reply:
Now they are gone.